ext/drreg/drreg.c

```diff
@@ -1155,21 +1956,10 @@ drreg_reservation_info(void *drcontext, reg_id_t reg, opnd_t *opnd OUT,
     return DRREG_SUCCESS;
 }
 
-drreg_status_t
-drreg_reservation_info_ex(void *drcontext, reg_id_t reg, drreg_reserve_info_t *info OUT)
+static void
+set_info(drreg_reserve_info_t *info, per_thread_t *pt, void *drcontext, reg_id_t reg,
```

Comment:
set_reservation_info or sthg: set_info is too generic

Reply:
Done.